hack/e2e-suite/liveness.sh

```diff
@@ -51,7 +51,7 @@ function teardown() {
 }
 
 function waitForNotPending() {
-  pod_id_list=$(${KUBECTL} get pods -o template '--template={{range.items}}{{.id}} {{end}}' -l test=liveness)
+  pod_id_list=$(${KUBECTL} get pods -o template '--template={{range.items}}{{.id}} {{end}}' -l'test in (liveness)')
```

Comment:
Why is this change required?
IIUC, both "test=liveness" and "test in (liveness)" will work, right?